src/main/scala/cromwell/util/google/GoogleCloudStorage.scala

```diff
@@ -38,11 +38,17 @@ case class GoogleCloudStorage(client: Storage) {
   def uploadObject(gcsPath: GoogleCloudStoragePath, fileContent: String): Unit = {
     val fileBytes = fileContent.getBytes
     val bais = new ByteArrayInputStream(fileBytes)
-    uploadObject(gcsPath, bais, fileBytes.length)
+    uploadObject(gcsPath, bais, fileBytes.length, "application/octet-stream")
   }
 
-  def uploadObject(gcsPath: GoogleCloudStoragePath, inputStream: InputStream, byteCount: Long): Unit = {
-    val mediaContent: InputStreamContent = new InputStreamContent("application/octet-stream", inputStream)
+  def uploadJson(gcsPath: GoogleCloudStoragePath, fileContent: String): Unit = {
+    val fileBytes = fileContent.getBytes
+    val bais = new ByteArrayInputStream(fileBytes)
+    uploadObject(gcsPath, bais, fileBytes.length, "application/json")
```

Comment:
It seems wasteful to have to uploadObjects identical except for the json vs octet-stream, can we abstract that out?